README.md

```diff
@@ -11,14 +11,14 @@
 
 ## Instructions
 
-- Download Tor browser from the official site ([link](https://www.torproject.org/download/download))
+- Download Tor using the following instructions: ([link](https://2019.www.torproject.org/docs/debian.html.en))
 
 - Configure your hidden service.
-    + Go to the Tor browser directory.
-    + Open the "torrc" file in an editor. (Probably located in `Browser/TorBrowser/Data/Tor` directory)
+    + Go to the tor directory.
+    + Open the "torrc" file in an editor. (Probably located in `/etc/tor/torrc` directory)
     + Add the following lines to the file
     ```
-    HiddenServiceDir /any/path/where/you/want/config/to/be/stored
+    HiddenServiceDir /var/lib/tor/hidden_service/
```

Comment:
Not saying it has to be, but it's the default path written in the Tor Project docs, and it has the appropriate permissions for tor already. Considering the fact that this repo is for beginners, it would be better to show the default path, just my opinion tho.